OneCommunityGlobal/HighestGoodNetworkApp
# Kaikanes config.js syntax bug

# Description
I noticed a console error
![Syntax error config js](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/a148054b-29fb-4833-8f0d-2c1d380ff148)
After looking into this I found there is no config.js file.
and our html is looking for one and not finding it causing the error
If there is suppose to be a config.js let me know

## Related PRS (if any):
Only frontend related


## Main changes explained:
- Commented out line 14 on public/index.html


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as anyone
4. open the console
5. verify the console error is gone

## Screenshots or videos of changes:

## Note:
If there is suppose to be a config.js let me know
